packages/http/src/forwarder/index.ts

```diff
@@ -12,7 +12,7 @@ const { version: prismVersion } = require('../../package.json');
 const forward: IPrismComponents<IHttpOperation, IHttpRequest, IHttpResponse, IHttpConfig>['forward'] = (
   resource: IHttpOperation,
   input: IHttpRequest,
-  timeout: number = 0,
```

Comment:
hm.. ? why remove this?

Reply:
This is an automatic fix of ESLint, I guess for some reason decided that inferred arguments are better. Will see if I can revert this.